### DISCUSSION TITLE
How to Create a Whatsapp Bot With Node.js — Jagad Yudha Awali

### DISCUSSION THREAD

**Discussion by giscus** [jagadyudha/jagad.dev#31] [Announcements]
# How to Create a Whatsapp Bot With Node.js — Jagad Yudha Awali

Using a third-party library to make a WhatsApp bot for free and customable.

https://jagad.dev/posts/how-to-create-a-whatsapp-bot-with-node-js

**Comment by jagadyudha** [comment]
Discussion and feedback are appreciated 😁

**Reply by prnv404** [reply]
Thanks bruh :)

**Reply by WritwikRay** [reply]
Hi there! excellent write-up but there's a typo..in the requirements it should be "npm install qrcode-terminal"...but anybody can just see it though...

**Comment by Rajaram-hash** [comment]
hii sir how to host this on render please explain

**Comment by BarrionuevoM** [comment]
i have problems installing the qr-code-terminall; the error says:
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/qr-code-terminall - Not found
npm ERR! 404
npm ERR! 404  'qr-code-terminall@*' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

**Reply by jagadyudha** [reply]
"npm install qrcode-terminal"

**Comment by hassanharoon321** [comment]
How we can deploy???

**Comment by daoudiyounes** [comment]
npm install --save https://github.com/Julzk/whatsapp-web.js/tree/jkr_hotfix_12
npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated rimraf@2.7.1: Rimraf versions prior to v4 are no longer supported
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated fstream@1.0.12: This package is no longer supported.
npm warn deprecated puppeteer@18.2.1: < 22.5.0 is no longer supported
